source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -115,7 +118,12 @@ export interface IEvaluationContext {
      * The root data object the template will bind to. Expressions that refer to $root in the template payload
      * map to this field. Initially, $data also maps to $root.
      */
-    $root: any
+    $root: any;
+    /**
+     * The host data object the template will bind to. Expressions that refer to $host in the template payload
+     * map to this field.
+     */
+    $host?: any;
```

Comment:
Forgot to mention -- with this addition, we should probably bump the `adaptivecards-templating` `package.json` version by one :)

Reply:
Bump major or minor?